README.md

```diff
@@ -759,6 +760,49 @@ See [javadocs](https://static.runelite.net/api/runelite-api/net/runelite/api/Gra
 
 </details>
 
+### Trades:
+
+`%COUNTERPARTY%` will be replaced with the name of the other user involved in the trade
+
+`%GROSS_VALUE%` will be replaced with the sum of item values offered by both parties in the transaction.
+
+`%NET_VALUE%` will be replaced with the value of the received items _minus_ the value of the given items.
+
+<details>
+  <summary>JSON for Player Trade Notifications:</summary>
+
+```json5
+{
+  "content": "%USERNAME% traded with %COUNTERPARTY%",
+  "type": "TRADE",
+  "playerName": "%USERNAME%",
+  "accountType": "NORMAL",
+  "extra": {
+    "counterparty": "%COUNTERPARTY%",
+    "receivedItems": [
+      {
+        "id": 314,
+        "quantity": 100,
+        "priceEach": 2,
+        "name": "Feather"
+      }
+    ],
+    "discardedItems": [
```

Comment:
Using the phrasing from the trade dialog itself, `receivedItems` is correct, and we would want to use `givenItems` instead of `discardedItems`

Reply:
trying to think of alternatives, `givenItems` has the downside that "You were given item xyz" suggests receive... but congruence with widget may be worth

`relinquishedItems` seems too formal

switching to `boughtItems`/`soldItems` might suggest a marketplace like GE rather than direct barter